src/index.tsx

```diff
@@ -115,10 +115,12 @@ export function configureAuth<
     children,
     renderLoading,
     renderUnauthenticated,
+    renderError = (status) => <div>Unhandled status: {status}</div>,
```

Comment:
Let's change it to the following to be able to use the error value:
`renderError = (error: Error) => <>{JSON.stringify(error)}</>,`